css/selectors/matches.json

```diff
@@ -8,60 +8,95 @@
           "support": {
             "chrome": [
               {
-                "version_added": "66"
+                "version_added": "66",
```

Comment:
It's actually 65. Opera has this flag too. https://www.chromestatus.com/features/5445716612743168